tokendito/helpers.py

```diff
@@ -410,25 +429,18 @@ def process_okta_credentials():
     set_okta_password()
 
 
-def process_okta_aws_app_url(app_url=None):
+def process_okta_aws_app_url():
     """Process Okta app url.
 
     :param app_url: string with okta tile URL.
     :return: None.
     """
-    if app_url is None:
-        logging.error(
-            "Okta Application URL not found in profile '{}'.\nPlease verify your options"
-            " or re-run this application with the --configure flag".format(settings.okta_profile))
-        sys.exit(2)
-    # Prepare final Okta and AWS app Url
-    url = urlparse(app_url)
-
-    if url.scheme == '' or url.netloc == '' or url.path == '':
-        logging.error("Okta Application URL invalid. Please check your configuration"
-                      " and try again.")
+    if not validate_okta_aws_app_url(input_url=settings.okta_aws_app_url):
```

Comment:
i would  just call like validate_okta_aws_app_url(settings.okta_aws_app_url).

Reply:
We are checking whether or not the URL is valid. `validate_okta_aws_app_url` returns a boolean, so we need to see whether the validation passed.

Reply:
ah! understood. removed the assignment during the function call.